### ISSUE TITLE
Support Shift+Delete / Shift + Insert in the editor as 2nd default

### ISSUE THREAD

**Issue by fade0ff** [GeorgRottensteiner/C64Studio#27]
Every editor I used in the last twenty years or more supported "Shift + Delete" to cut text into the clipboard and "Shift+Insert" to paste text from the clipboard. I figured this was a a de facto standard that has its roots in early terminal simulations, made it into the IBM Common User Access standard and is one of the two default key bindings (in addition to Ctrl+X / Ctrl+V) in Windows from the very beginning.
So it feels crippling to use the editor without these key bindings. I'm aware that I can redefine the key bindings but this way I lose the other default binding (Ctrl+X / Ctrl+V) which I use as well depending on the situation.
In a nutshell: would it be possible to support both default key bindings? Or at least let users define more than one key binding for a function?

**Comment by GeorgRottensteiner** [comment]
Sounds feasible and shouldn't be too hard, I'll see to it!

**Comment by GeorgRottensteiner** [comment]
I've checked in an update with commit 25c17ab5. Now you can set a secondary key binding. On a first simple test that works fine with both key bindings for copy/paste.

Have fun and let me know if there are any side effects. On a short test run I didn't run into any though.

**Comment by fade0ff** [comment]
From what I can say, this works just as expected (used it for Cut, Insert, Undo, Redo). Thanks for the incredible support!

**Comment by GeorgRottensteiner** [comment]
Great! Closing this then
